packages/cli/src/rpc/navie/models/handlers/getConfig.ts

```diff
@@ -0,0 +1,38 @@
+import { NavieRpc } from '@appland/rpc';
+import { RpcHandler } from '../../../rpc';
+
+/**
+ * This function is used to print a secret value as asterisks. If the value doesn't exists,
+ * it returns undefined.
+ * @param value the secret value
+ * @returns the secret value as asterisks or undefined
+ */
+const printSecret = (value?: string) => (value?.length ? '*'.repeat(value.length) : undefined);
```

Comment:
IMO it should just print a constant number of asterisks. Not that it matters terribly here ;) but a secret's length is also secret! (Also, a nit: this doesn't actually print anything ;p)